### PR TITLE
OSDOCS-6510: remove dual stack and secondary gateway interface

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13+

Issues:
https://issues.redhat.com/browse/OSDOCS-6510
https://issues.redhat.com/browse/OSDOCS-6079

Link to docs preview:
https://61255--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-networking.html#microshift-network-features_microshift-networking

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@jogeo 
And @pliurh Could you both check, please? 
I removed the dual stack reference and the secondary gateway text; can you confirm that can all be removed, @pliurh? Thank you!

**Comment by jogeo** [comment]
Thanks Shauna, the dual stack reference I was referring to is gone.

**Comment by pliurh** [comment]
@ShaunaDiaz I can still see `externalGatewayInterface` in the preview doc. It shall be removed.

**Comment by ShaunaDiaz** [comment]
acf47bb complete scrub of `externalGatewayInterface` from yaml samples and table

**Comment by pliurh** [comment]
LGTM

Only curious about why we comment out those paragraphs instead of deleting them.

**Comment by ShaunaDiaz** [comment]
> LGTM
> 
> Just curious about why we comment out those paragraphs instead of deleting them.

@pliurh I wanted you to see what was being removed.  I will delete them now.

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* A cluster installed on infrastructure configured with the Open Virtual Network (OVN)-Kubernetes network plugin.
+* Access to the `kubeconfig` file.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+Run the commands listed in each step that follows to restore the `NodePort` service traffic after you install{product-title}:
```
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:20]: Per the Modular documentation guidelines, the "Procedure" section should only contain the steps. You can remove this sentence which repeats what is in the procedure introduction.

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+. Find the name of the ovn-master pod that you want to restart by running the following command:
```
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:22]:
```suggestion
. Find the name of the `ovn-master` pod that you want to restart by running the following command:
```

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
++
+[source, terminal]
+----
+$ pod=$(oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master | awk -F " " '{print $1}')
+----
+
+. Force a restart of the of the ovnkube-master pod by running the following command:
```
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:29]:
```suggestion
. Force a restart of the of the `ovnkube-master` pod by running the following command:
```

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+$ oc -n openshift-ovn-kubernetes delete pod $pod
+----
+
+. Optional: To confirm that the ovnkube-master pod restarted, run the following command:
++
+[source, terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes
+----
+If the pod restarted, the listing of the running pods shows a different ovnkube-master pod name and age consistent with the procedure you just completed.
```
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:42]: Should you include an Example output here?

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+----
+
+. Force a restart of the of the ovnkube-master pod by running the following command:
++
+[source, terminal]
+----
+$ oc -n openshift-ovn-kubernetes delete pod $pod
+----
+
+. Optional: To confirm that the ovnkube-master pod restarted, run the following command:
```
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:36]: Suggest moving steps 3 and 4 under a `.Verification` heading

Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:36]:
```suggestion
. To confirm that the `ovnkube-master` pod has restarted, run the following command:
```

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc (hunk 4 above)
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:42]:
```suggestion
+
If the pod has restarted, the list of running pods shows the `ovnkube-master` pod name and age consistent with the procedure you just completed.
```

File: _unused_topics/microshift-nodeport-unreachable-workaround.adoc [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
++
+[source, terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes
+----
+If the pod restarted, the listing of the running pods shows a different ovnkube-master pod name and age consistent with the procedure you just completed.
+
+. Verify that the `NodePort` service can now be reached.
```
Comment (sjhala-ccs) [inline _unused_topics/microshift-nodeport-unreachable-workaround.adoc:44]: Is there an example command that can go here?

**Review by sjhala-ccs — COMMENTED** [review]
A few suggestions for you to consider, thanks Shauna!

**Comment by ShaunaDiaz** [comment]
> A few suggestions for you to consider, thanks Shauna!

Hey @sjhala-ccs I have not made any suggested corrections because this module was moved to `unused_topics` meaning we aren't using it. It was essentially removed from the active repo.

This PR just removes content overall.

**Comment by sjhala-ccs** [comment]
@ShaunaDiaz Cherry picked failed for both 4.13 and 4.14 which is unusual. Maybe there is a previous PR that was not merged properly. @bergerhoffer can you please check? Thank you!

**Comment by bergerhoffer** [comment]
@sjhala-ccs Thanks for tagging me to check on this!

@ShaunaDiaz it looks like main and 4.13/4.14 got out of sync by this manual cherry pick: https://github.com/openshift/openshift-docs/pull/57484

The original PR (https://github.com/openshift/openshift-docs/pull/57170) doesn't do anything with the additional resources section in microshift-networking.adoc, but the manual CP removes the section all together. It looks like you then tried to sync main and 4.13 by https://github.com/openshift/openshift-docs/pull/57438 but that was for changes in an older PR. So it got out of sync again :)

So please do go ahead with the manual CPs for 4.13 and 4.14, and add the additional resource section back in. I think that hopefully that will get you back in sync again.

If you're unsure whether that'll get you totally in sync or not, you can always check out the file in the two different branches (main vs 4.14) and do a comparison on them to be sure.

**Pull request merged**